packages/date-time/src/components/pages/CalendarPage.tsx

```diff
@@ -115,13 +102,24 @@ export class CalendarPage extends React.Component<{}, {}> {
               <CalendarInlineExample
                 dateRangeType={DateRangeType.WorkWeek}
                 firstDayOfWeek={DayOfWeek.Monday}
-                autoNavigateOnSelection={true}
                 highlightCurrentMonth={false}
                 highlightSelectedMonth={true}
                 showGoToToday={true}
                 workWeekDays={[DayOfWeek.Tuesday, DayOfWeek.Saturday, DayOfWeek.Wednesday, DayOfWeek.Friday]}
               />
             </ExampleCard>
+            <ExampleCard
+              title="Calendar with multiday view using dateRangeType === DateRangeType.Day and daysToSelectInDayView = 4"
+              code={CalendarInlineExampleCode}
+            >
+              <CalendarInlineExample
+                dateRangeType={DateRangeType.Day}
+                highlightCurrentMonth={false}
+                highlightSelectedMonth={true}
+                showGoToToday={true}
+                calendarDayProps={{ daysToSelectInDayView: 4 }}
```

Comment:
nit: could we add a quick selector to allow users to change the number of days to select on the demo page?